pycbc/inference/models/base.py

```diff
@@ -736,9 +749,12 @@ def _init_args_from_config(cls, cp):
         except ValueError:
             sampling_transforms = None
         args['sampling_transforms'] = sampling_transforms
-        # get any other keyword arguments provided
-        args.update(
-                cls.extra_args_from_config(cp, section, skip_args=['name']))
+        # get any waveform transforms
+        if any(cp.get_subsections('waveform_transforms')):
+            logging.info("Loading waveform transforms")
+            args['waveform_transforms'] = \
```

Comment:
The docstring for this function above (lines 712 and 723) would also need to be updated saying waveform transforms are also being retrieved.

Reply:
Ok, fixed.